src/components/_modal/pages/bookmarkOrder/multi-order/checkoutForm/MultiOrderCheckoutForm.js

```diff
@@ -0,0 +1,223 @@
+import Title from "@/components/base/title";
+import Text from "@/components/base/text";
+import { IconLink } from "@/components/base/iconlink/IconLink";
+import ChevronRight from "@/public/icons/chevron_right.svg";
+import cx from "classnames";
+import localizationStyles from "./Localization.module.css";
+import styles from "./MultiOrderCheckoutForm.module.css";
+import Translate from "@/components/base/translate";
+import OrdererInformation from "../../../order/ordererinformation/OrdererInformation";
+import { onMailChange } from "@/components/_modal/pages/order/utils/order.utils";
+import { useMemo, useState } from "react";
+import useOrderPageInformation from "@/components/hooks/useOrderPageInformations";
+import Button from "@/components/base/button";
+import { useModal } from "@/components/_modal/Modal";
+import { LOGIN_MODE } from "../../../login/utils";
+
+const LocalizationInformation = ({
```

Comment:
Du har nok set order --> localizationinformation. 
var det for svært at flytte den her og orders localization information sammen?

Reply:
er der flere ting end onClick & iconLink texten der er forskellige fra single-order localization info? Ellers kunne man give de to ting som parameter....

Reply:
Ville rigtig gerne, men kan gennemskue implicationerne ved at refaktorere LocalizationInformation til at kunne understøtte den mere simple brug. Jeg kan godt give det et bud

Reply:
har prøvet at bruge den ved at gå udenom dens wrapper